build.xml

```diff
@@ -121,6 +123,14 @@
   <target name="-real-build">
     <mkdir dir="${build-src.dir}"/>
 
+    <!-- Make sure that the bootclasspath isn't newer than expected. -->
```

Comment:
If someone has only JDK 8 on their machine, they set JAVA_HOME, and then they run ant (without setting more shell variables or ant properties) will compilation proceed for that individual?

Reply:
This is case 1. They will get the javac warning, but the build will succeed.

warning: [options] bootstrap class path not set in conjunction with -source 1.6